doc/htm.md

```diff
@@ -0,0 +1,37 @@
+HTM Indexing
+============
+
+Overview                        {#htm-overview}
+========
```

Comment:
So yes, using `----` for a section breaks doxygen output if a label is present ([see this stackoverflow post](http://stackoverflow.com/questions/13648124/header-label-in-doxygen-markdown-page-makes-header-title-disappear)).